test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -58,6 +58,7 @@ private static int Run(Options opts)
                 if (opts.WarmUp)
                 {
                     var (_, outputPath, repositoryPath, docfxConfig) = Prepare(opts, workingFolder, remoteBranch);
+                    Clean(outputPath);
```

Comment:
Is this needed? `Cleanup` will be called while running test anyway.

Reply:
In case we want to re-run the warm-up manually.